src/api/spec/controllers/source_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
+# WARNING: If you change owner tests make sure you uncomment this line
+# and start a test backend. Some of the Owner methods
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
```

Comment:
Does this comment apply to all controller specs? Maybe it should go in the test suite guide instead?
https://github.com/evanrolfe/open-build-service/blob/master/src/api/spec/README.md

Also what are the "owner tests" and "owner methods"?